administrator/components/com_languages/tmpl/language/edit.php

```diff
@@ -71,6 +61,27 @@
 		<?php echo $this->form->renderFieldset('site_name'); ?>
 		<?php echo JHtml::_('bootstrap.endTab'); ?>
 
+		<?php echo JHtml::_('bootstrap.addTab', 'myTab', 'automatic_association', JText::_('COM_LANGUAGES_FIELDSET_AUTOMATIC_ASSOCIATION_LABEL')); ?>
```

Comment:
There should be here a conditional checking if `Associations` is enabled in the Languagefilter plugin

Reply:
And I guess also for the new Fallback Language column in Content Languages Manager